screens/DoubleSearch.js

```diff
@@ -1,10 +1,10 @@
-import React, { useEffect } from "react";
+import React, { useEffect, useRef } from "react";
```

Comment:
<img width="749" alt="Screen Shot 2020-04-05 at 5 44 37 PM" src="https://user-images.githubusercontent.com/35850933/78510678-2129a100-7765-11ea-9890-5f79bcb8658f.png">

Reply:
This was something I removed in my PR yesterday, its showing up again I guess because when you guys started working on this issue the branch wasn't up to date

Reply:
yeee exactly!